schedule/qam/15-SP1/qam-minimal-base@64bit.yaml

```diff
@@ -62,4 +61,18 @@ schedule:
 - console/postgresql_server
 - console/orphaned_packages_check
 - console/coredump_collect
+conditional_schedule:
+  boot:
+    MACHINE:
+      zkvm:
+        - installation/bootloader_zkvm
+      64bit:
+        - installation/isosize
+        - installation/bootloader
```

Comment:
It's kind of late, but there are some hints I can provide as we solved some of the issue.
* check if different modules make sense, `isosize` can be easily dropped. 
* `tests/installation/bootloader_start.pm` can be used as a wrapper for all bootloaders, no need for conditions
* For the differences you cannot avoid (which are machine specific) I would recommend to create separate schedule and override it in the job group (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md#4-use-different-schedules-for-the-same-scenario)

Reply:
I was looking that isosize is somewhere scheduled and somewhere not, but as the test takes 0 seconds to run it will not harm, but maintenance is not modifying ISO so I think I can remove it and use the wrapper. thx @rwx788

Reply:
@rwx788 tried to use `tests/installation/bootloader_start.pm`, but it was strangely [failing](http://10.100.12.155/tests/13689) and this wrapper is not used except one yaml profile which is also not used. :roll_eyes:

Reply:
I can see that in your failing example you have `bootloader_zkvm`

Reply:
Yes, thats the problem that for some reason the `tests/installation/bootloader_start.pm` does schedule `bootloader_zkvm` on x86_64 test

```
---
name: mau-filesystem
description: Filesystem tests on Maintenance prepared image 
vars:
  COLLECT_COREDUMPS: '1'
schedule:
- installation/bootloader_zkvm
- boot/boot_to_desktop
- console/prepare_test_data
- console/consoletest_setup
- console/system_prepare
- console/lsof
- console/autofs
- console/btrfs_autocompletion
- console/btrfs_qgroups
- console/snapper_cleanup
- console/btrfs_send_receive
- console/snapper_undochange
- console/snapper_create
- console/snapper_thin_lvm
- console/snapper_used_space
- console/coredump_collect
...
```